cpp/basix/cwrapper.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022 Susanne Claus
```

Comment:
Could you add an example of use in e.g. `demo/c/wrapper`?

That's often helpful for getting the API looking right.